src/data-dictionary/events/ContainerSample/memorySwapLimitBytes.md

```diff
@@ -0,0 +1,9 @@
+---
+name: memorySwapLimitBytes
+type: attribute
+units: bytes (B)
+events:
+  - ContainerSample
+---
+
+The total amount of memory+swap the container is allowed to use.
```

Comment:
I would avoid the "arithmetic" expression in the description, and instead word it like:

> The total amount of memory the container is allowed to use, including swap

But it's just a personal preference